ethereum/core/src/main/java/tech/pegasys/teku/core/blockvalidator/BatchSignatureVerifier.java

```diff
@@ -68,6 +74,10 @@ public synchronized boolean verify(
    * instance methods would fail with exception
    */
   public synchronized boolean batchVerify() {
+    if (!BLSConstants.VERIFICATION_ENABLED) {
+      LOG.warn("Skipping bls verification.");
+      return true;
+    }
```

Comment:
I would localize this flag usage inside BLS module only. 
You may alter ` BLS.prepareBatchVerify` and `BLS.completeBatchVerify` behavior based on this flag

Reply:
Decided to skip on this as discussed offline.

Reply:
I've localized the flag but kept the `BLS.prepareBatchVerify` functionality the same.